app/core/search.py

```diff
@@ -391,6 +391,7 @@ def _process_vespa_search_response_families(
                 or hit.family_category is None
                 or hit.family_source is None
                 or hit.family_geography is None
```

Comment:
**question:** If this was replaced above, could it be removed from here?

Reply:
I'm not sure I follow! This is a check as to whether the `hit` that is returned from vespa has all the relevant information to construct a `SearchResponseFamily` object.